.github/workflows/build_and_test.yml

```diff
@@ -217,6 +217,11 @@ jobs:
       run: |
         python3.6 -m pip install numpy 'pyarrow<3.0.0' pandas scipy xmlrunner plotly>=4.8
         python3.6 -m pip list
+    - name: List Python packages (Python 3.9)
+      run: |
+        curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+        python3.9 get-pip.py
```

Comment:
Can we remove it back now, since it's fixed?

Reply:
Certainly, removed.